src/server/src/main/java/io/cassandrareaper/core/RepairUnit.java

```diff
@@ -42,7 +43,9 @@ private RepairUnit(Builder builder, UUID id) {
     this.clusterName = builder.clusterName;
     this.keyspaceName = builder.keyspaceName;
     this.columnFamilies = builder.columnFamilies;
-    this.incrementalRepair = builder.incrementalRepair;
+    // If subrange incremental repair is enabled, incremental repair must also be enabled
+    this.incrementalRepair = builder.incrementalRepair || builder.subrangeIncrementalRepair;
```

Comment:
comment doesn't match.

can this be enforced at the config layer ?

Reply:
I'll update the comment, I ended up making it seamless. You can enable either or any of these flags, it will run an incremental repair.

Reply:
Also, from the UI perspective this is transparent as we now select a "repair type" which can be either of the three types. No need to select both incremental and subrange incremental there.